addons/udes_stock/models/stock_location.py

```diff
@@ -801,6 +833,11 @@ def button_view_child_locations(self):
             "context": {"default_location_id": self.id},
         }
 
+    def update_scanning_date_start(self):
+        """Method to be override in other modules"""
+        self.ensure_one()
+        return None
```

Comment:
Question: Do we really need to return?

Reply:
There are some dummy methods which are merged that return None. Me personally, prefer to return smth

Reply:
We can remove the return None statement but it can't leave it